### ISSUE TITLE
Error tab detection no longer works

### ISSUE THREAD

**Issue by kaedroho** [torchbox/wagtail#2380]
When a validation error occurs, a red badge should be pinned to any tab which contains a validation error. This no longer seems to work on Wagtail 1.4.1.

The following error displays in the Javascript console:
```
Error: Syntax error, unrecognized expression: .tab-nav a[href=#content]
```

Which I think corresponds to the following line of code:
https://github.com/torchbox/wagtail/blob/5c0063f429aea44a20d0b82989f7ca8db8ab899e/wagtail/wagtailadmin/static_src/wagtailadmin/js/page-editor.js#L343

**Comment by gasman** [comment]
My guess is that's a result of us bumping jQuery to 2.2. (Maybe it now requires the href value to be in quotes?)

**Comment by kaedroho** [comment]
Yep, that seems to fix both this and #2381
